### ISSUE TITLE
Protect from systems without notification server

### ISSUE THREAD

**Issue by kynikos** [kynikos/outspline#376]
If a notification server is not installed, Outspline will crash even if `libnotify` is installed.
Perhaps also write a reminder in the optdepends `libnotify` entry.

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/outspline/coreaux/__init__.py", line 96, in run_with_excepthook
    run_old(*args, **kw)
  File "/usr/lib/python2.7/threading.py", line 1073, in run
    self.function(*self.args, **self.kwargs)
  File "/usr/lib/python2.7/site-packages/outspline/extensions/organism_timer/timer.py", line 555, in _activate_occurrences_block
    self._activate_occurrences(time, occsd)
  File "/usr/lib/python2.7/site-packages/outspline/extensions/organism_timer/timer.py", line 559, in _activate_occurrences
    activate_occurrences_event.signal(time=time, occsd=occsd)
  File "/usr/lib/python2.7/site-packages/outspline/coreaux/events.py", line 40, in signal
    handler(kwargs)
  File "/usr/lib/python2.7/site-packages/outspline/extensions/organism_alarms/__init__.py", line 214, in _handle_activate_occurrences
    occsd[filename])
  File "/usr/lib/python2.7/site-packages/outspline/extensions/organism_alarms/alarmsmod.py", line 182, in activate_alarms
    self._activate_alarm(occ)
  File "/usr/lib/python2.7/site-packages/outspline/extensions/organism_alarms/alarmsmod.py", line 213, in _activate_alarm
    alarm=alarm['alarm'])
  File "/usr/lib/python2.7/site-packages/outspline/coreaux/events.py", line 40, in signal
    handler(kwargs)
  File "/usr/lib/python2.7/site-packages/outspline/plugins/notify/__init__.py", line 88, in _handle_alarm
    self.alarm.show()
Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Notifications was not provided by any .service files (2)
```

**Comment by kynikos** [comment]
Fix together with #125.

**Comment by kynikos** [comment]
Fixed, #125 will wait 0.9.0
